### PR TITLE
Add French Version of the Charter

### PR DESCRIPTION
**Pull request by Hornan7**
- Translation work.
- Reference to the original document includes the revision number.

### CODE REVIEW

**Review by thiago-tsnnst — APPROVED** [review]

File: README.fr.md [hunk 1]
```diff
@@ -0,0 +1,297 @@
...
+long terme de la blockchain Cardano.
+
+À mesure que nous avançons, nous encourageons un dialogue ouvert, un apprentissage continu et 
+un engagement proactif avec la communauté au sens large. Grâce à nos efforts combinés, 
+nous pouvons garantir que le Conseil atlantique de Cardano reste une composante vitale et efficace 
+du comité constitutionnel de Cardano, défendant les intérêts et 
+les principes de l'ensemble de l'écosystème de Cardano.
+
+## Signé et accepté par :
+[Mike Hornan](https://x.com/Hornan7)
```
Comment (Crypto2099) [inline README.fr.md:297]:
```suggestion
[Mike Hornan](https://x.com/Hornan7)
[Adam Dean](https://x.com/adamkdean)
```

File: docs/README.fr.md [hunk 2]
```diff
@@ -0,0 +1,297 @@
...
+long terme de la blockchain Cardano.
+
+À mesure que nous avançons, nous encourageons un dialogue ouvert, un apprentissage continu et 
+un engagement proactif avec la communauté au sens large. Grâce à nos efforts combinés, 
+nous pouvons garantir que le Conseil atlantique de Cardano reste une composante vitale et efficace 
+du comité constitutionnel de Cardano, défendant les intérêts et 
+les principes de l'ensemble de l'écosystème de Cardano.
+
+## Signé et accepté par :
+[Mike Hornan](https://x.com/Hornan7)
```
Comment (Crypto2099) [inline docs/README.fr.md:297]:
```suggestion
[Mike Hornan](https://x.com/Hornan7)
[Adam Dean](https://x.com/adamkdean)
```

**Review by Crypto2099 — APPROVED** [review]
Thank you @Hornan7 for providing this French translation!

File: README.fr.md [hunk 3]
```diff
@@ -0,0 +1,298 @@
...
+[Adam Dean](https://x.com/adamkdean)
```
Comment (Hornan7) [inline README.fr.md:298]:
```suggestion
[Adam Dean](https://x.com/adamkdean)
[Thiago Nunes](https://x.com/tsnnst)
```

File: docs/README.fr.md [hunk 4]
```diff
@@ -0,0 +1,298 @@
...
+[Adam Dean](https://x.com/adamkdean)
```
Comment (Hornan7) [inline docs/README.fr.md:298]:
```suggestion
[Adam Dean](https://x.com/adamkdean)
[Thiago Nunes](https://x.com/tsnnst)
```

**Review by Hornan7 — COMMENTED** [review]
Received Verbal approval from @thiago-tsnnst

File: README.fr.md [hunk 5]
```diff
@@ -0,0 +1,299 @@
...
+[Thiago Nunes](https://x.com/tsnnst)
```
Comment (rickymac68) [inline README.fr.md:299]:
```suggestion
[Thiago Nunes](https://x.com/tsnnst)
[Richard McCracken](https://x.com/RichardMcCrackn)```

File: docs/README.fr.md [hunk 6]
```diff
@@ -0,0 +1,299 @@
...
+[Thiago Nunes](https://x.com/tsnnst)
```
Comment (rickymac68) [inline docs/README.fr.md:299]:
```suggestion
[Thiago Nunes](https://x.com/tsnnst)
[Richard McCracken](https://x.com/RichardMcCrackn)
```

**Review by rickymac68 — APPROVED** [review]
Reviewed and signed by Richard McCracken

File: docs/README.fr.md [hunk 7]
```diff
@@ -0,0 +1,300 @@
...
+[Richard McCracken](https://x.com/RichardMcCrackn)
```
Comment (Hornan7) [inline docs/README.fr.md:300]:
```suggestion
## Traduction signé et accepté par :
[Mike Hornan](https://x.com/Hornan7)
[Adam Dean](https://x.com/adamkdean)
[Thiago Nunes](https://x.com/tsnnst)
[Richard McCracken](https://x.com/RichardMcCrackn)

## Approbation basée sur le document original :
[Beatrice Anihiri](https://x.com/BeatriceAnihiri)
[Jenny Brito](https://x.com/Jennycitalinda)

```

File: README.fr.md [hunk 8]
```diff
@@ -0,0 +1,300 @@
...
+[Richard McCracken](https://x.com/RichardMcCrackn)```
```
Comment (Hornan7) [inline README.fr.md:300]:
```suggestion
## Traduction signé et accepté par :
[Mike Hornan](https://x.com/Hornan7)
[Adam Dean](https://x.com/adamkdean)
[Thiago Nunes](https://x.com/tsnnst)
[Richard McCracken](https://x.com/RichardMcCrackn)```

## Approbation basée sur le document original :
[Beatrice Anihiri](https://x.com/BeatriceAnihiri)
[Jenny Brito](https://x.com/Jennycitalinda)
```

**Review by Hornan7 — COMMENTED** [review]
Adding approval based on original document

File: README.fr.md (hunk 8 above)
Comment (Hornan7) [inline README.fr.md]:
```suggestion
[Mike Hornan](https://x.com/Hornan7)
[Adam Dean](https://x.com/adamkdean)
[Thiago Nunes](https://x.com/tsnnst)
[Richard McCracken](https://x.com/RichardMcCrackn)
[Beatrice Anihiri](https://x.com/BeatriceAnihiri)
[Jenny Brito](https://x.com/Jennycitalinda)
```

File: docs/README.fr.md (hunk 7 above)
Comment (Hornan7) [inline docs/README.fr.md:300]:
```suggestion
[Mike Hornan](https://x.com/Hornan7)
[Adam Dean](https://x.com/adamkdean)
[Thiago Nunes](https://x.com/tsnnst)
[Richard McCracken](https://x.com/RichardMcCrackn)
[Beatrice Anihiri](https://x.com/BeatriceAnihiri)
[Jenny Brito](https://x.com/Jennycitalinda)
```

**Pull request merged**